src/main/java/org/capstone/job_fair/models/entities/attendant/cv/CvEntity.java

```diff
@@ -0,0 +1,92 @@
+package org.capstone.job_fair.models.entities.attendant.cv;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+@Entity
+@Getter
+@Setter
+@ToString
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name = "cv", schema = "dbo")
+public class CvEntity {
+    @Id
+    @Column(name = "id")
+    private String id;
+    @Basic
+    @Column(name = "firstname")
+    private String firstname;
+    @Basic
+    @Column(name = "lastname")
+    private String lastname;
+    @Basic
+    @Column(name = "middlename")
+    private String middlename;
+    @Basic
+    @Column(name = "year_of_exp")
+    private Double yearOfExp;
+    @Basic
+    @Column(name = "email")
+    private String email;
+    @Basic
+    @Column(name = "cell_number")
+    private String cellNumber;
+    @Basic
+    @Column(name = "dob")
+    private Long dob;
+    @Basic
+    @Column(name = "marital_status")
+    private Boolean maritalStatus;
+    @Basic
+    @Column(name = "address")
+    private String address;
+    @Basic
+    @Column(name = "summary")
+    private String summary;
+    @Basic
+    @Column(name = "create_date")
+    private Long createDate;
+
```

Comment:
It's missing relationship with nationality, country, current_job_level_id and gender right?